### PR TITLE
feat(HMS-4758): display idmsvc in prod

### PR DESCRIPTION
**Pull request by avisiedo**
This change copy the configuration from stage to prod to make the Directory and Domain services available in production.

NOTE: This depends on promoting frontend and backend to production.

TODO:

- [x] Update metadata to display idmsvc in the left panel.

https://issues.redhat.com/browse/HMS-4758

### CODE REVIEW

File: static/stable/prod/navigation/settings-navigation.json [hunk 1]
```diff
@@ -88,6 +88,30 @@
...
           "href": "/settings/learning-resources"
-      }
+      },
+      {
+          "id": "idmsvc",
+          "appId": "idmsvc",
+          "title": "Directory and Domain Services",
+          "description": "Register identity and access systems to enable machines to automatically join a domain",
+          "href": "/settings/idmsvc",
+          "icon": "PlaceholderIcon",
```
Comment (frasertweedale) [inline static/stable/prod/navigation/settings-navigation.json:98]: I think we need also to restrict this to Preview mode, as we did in stage.  See PR https://github.com/RedHatInsights/chrome-service-backend/pull/674

File: static/stable/prod/main.yml [hunk 2]
```diff
@@ -103,6 +103,12 @@ content-sources:
     versions:
       - v1
 
+idmsvc:
+  title: Directory and Domain services
```
[the remark this answers is not in the document]
Comment (frasertweedale) [reply static/stable/prod/main.yml:107]: I think it's fine.

**Review by frasertweedale — APPROVED** [review]

**Comment by avisiedo** [comment]
@RedHatInsights/platform-experience hi mates! we don't need to merge, but we would need to have everything on place at the moment that all related MR/PR start to merge to go to production.

Did you have a chance to review the pr? if everything is ok, please let us know to be sure no additional changes would be required on this repo. thank you very much :smiley:

**Comment by frasertweedale** [comment]
> @avisiedo I'd strongly recommend using feature flags. We can merge and deploy the changes whenever we need and you can turn on/off the feature when ever you need. You won't have to coordinate with us.

We have decided to go with the current setup, and will consider converting to a feature flag while we evaluate the success of the feature in production.

This PR is now ready for final review and (hopefully) merge.

**Comment by frasertweedale** [comment]
ping @Hyperkid123 our service is deployed in prod and we are ready to proceed with this change.  Are you able to ack it, or should we nudge someone else?  Thanks!

**Review by Hyperkid123 — APPROVED** [review]

**Comment by Hyperkid123** [comment]
Once the image is build prod should be updated. it might take longer tna usual because Konflux is having hard time with release lately

**Pull request merged**